base/strings/string.jl

```diff
@@ -29,16 +29,28 @@ const ByteArray = Union{CodeUnits{UInt8,String}, Vector{UInt8},Vector{Int8}, Fas
 
 @inline between(b::T, lo::T, hi::T) where {T<:Integer} = (lo ≤ b) & (b ≤ hi)
 
+"""
+    String <: AbstractString
+
+The generic, default string type in Julia, used by e.g. string literals.
```

Comment:
Are you sure about the word "generic"? To me this feels as if you want to state that its actually a parametric type (i.e. `String{T}`)

Reply:
Yeah, I agree that it could be interpreted like that (although the intention here is more like "general-purpose").

It looks like the sentence work without "generic"?

```suggestion
The default string type in Julia, used by e.g. string literals.
```